src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -407,15 +405,12 @@ public override void VisitConstructorDeclaration(ConstructorDeclarationSyntax no
             }
         }
 
-        public override void VisitRecordDeclaration(RecordDeclarationSyntax node)
+        public override void VisitPrimaryConstructorBaseType(PrimaryConstructorBaseTypeSyntax node)
         {
-            Debug.Assert(node.ParameterList is object);
-            Debug.Assert(node.IsKind(SyntaxKind.RecordDeclaration));
-
-            if (node.PrimaryConstructorBaseTypeIfClass is PrimaryConstructorBaseTypeSyntax baseWithArguments)
-            {
-                VisitNodeToBind(baseWithArguments);
-            }
+            SyntaxNode previousNodeToBind = _nodeToBind;
+            _nodeToBind = node;
```

Comment:
>_nodeToBind = node; [](http://example.com/codeflow?start=12&length=19)

Is this necessary? It looks like ```FindExpressionVariables``` calls ```VisitNodeToBind``` on the root node and it doesn't look like manual adjustments are common for ```_nodeToBind```.

Reply:
I was just following the comment in the switch I added to that said everything in the switch needed a visit method. I don't see any behavior difference when removing it though.